packages/auth/src/core/auth/initialize.ts

```diff
@@ -65,9 +71,37 @@ export function initializeAuth(app: FirebaseApp, deps?: Dependencies): Auth {
 
   const auth = provider.initialize({ options: deps }) as AuthImpl;
 
+  if (_isFirebaseServerAppImpl(app)) {
+    if (app.authIdToken !== undefined) {
+      void _loadUserFromIdToken(auth, app.authIdToken);
+    }
+  }
+
   return auth;
 }
 
+export async function _loadUserFromIdToken(
+  auth: Auth,
+  idToken: string
+): Promise<void> {
+  try {
+    const response = await getAccountInfo(auth, { idToken });
+    const authInternal = _castAuth(auth);
+    await authInternal._initializationPromise;
+    const user = await UserImpl._fromGetAccountInfoResponse(
+      authInternal,
+      response,
+      idToken
+    );
+    await authInternal._updateCurrentUser(user);
```

Comment:
Could there be a race condition here? `_initializeAuth` also calls another hanging promise, `provider.initialize` which I think calls `_initializeAuthInstance` which eventually tries to check for a user in persistent storage and will try to `_updateCurrentUser` if it finds anything?

Reply:
FirebaseServerApps cannot be created in Browser environments. Is there persistence outside of browser environments?

I did some testing and printed out the persistence array in `_initializeAuthInstance` and it appears to be empty.

Reply:
It doesn't seem to, still I think it should have a more definitive implementation. I'll work on this  topic on Friday.

Reply:
Ok, I now suppress loading users from persistence upon Auth init for FirebaseServerApps.